src/seedu/addressbook/AddressBook.java

```diff
@@ -1161,8 +1161,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replaceFirst("^" + prefix, "");
```

Comment:
Does this method replaces the value of prefix only when it occurs at the beginning of fullString or it just replaces the first occurrence of the prefix in fullString? 

Methods can be more abstract.